dom/events.ts

```diff
@@ -207,6 +214,66 @@ export function trigger (element : null|EventTarget, type : string, data : any =
 }
 
 
+/**
+ * Adds media query listener
+ */
+export function mediaQuery (
```

Comment:
```suggestion
export function matchMedia (
```

Reply:
I'll call it `matchMediaQuery()` due to problems related to the `window. matchMedia()` functions.